next.config.js

```diff
@@ -0,0 +1,11 @@
+require('dotenv').config()
```

Comment:
I thought next.js already have a way to load env var automatically, without changes in next.config?

https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser

Reply:
Yep was going to comment about this too. do you mind removing this?

Reply:
ah correct 💯, removed dotenv package

Reply:
thanks

Reply:
What about use pigeon-maps instead raw mapbox ? No need api key
https://github.com/mariusandra/pigeon-maps